CLI/test/ressources/value_clauses/input/ValueClauses.rdz.tcbl

```diff
@@ -0,0 +1,34 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. DVZZMFT0.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+      
+       01 StructureType typedef strict PRIVATE.
+          05 champ PIC X(1).
+      
+       01 ScalarType typedef strict PRIVATE PIC X(1).
+      
+       01 C-DatePivot   TYPE Date VALUE '20190101'.
+       01 C-DefaultRate TYPE Rate VALUE 3.55.
+      
+       01 Structure TYPE StructureType     VALUE 'A'.
```

Comment:
Can you use `01 Structure VALUE 'A' TYPE StructureType.` ?
The goal is to check that the generated line doesn't include `TYPE StructureType`